app/src/main/res/layout/repository_list_view.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout android:id="@+id/pinnedLayout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:padding="10dp"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+
+
+    <TextView xmlns:android="http://schemas.android.com/apk/res/android"
+        android:id="@+id/repository"
+        android:layout_width="fill_parent"
+        android:layout_height="fill_parent"
+        android:textSize="16dip"
```

Comment:
Use `dp` to remain consistent. The documentation recommends using `sp` for text size.